frontend/package.json

```diff
@@ -2,71 +2,80 @@
     "name": "frontend",
     "version": "0.1.0",
     "private": true,
+    "type": "module",
     "dependencies": {
-        "@aldabil/react-scheduler": "^2.7.26",
-        "@aws-amplify/auth": "^5.6.4",
+        "@aldabil/react-scheduler": "^2.8.8",
+        "@aws-amplify/auth": "^5.6.12",
         "@emotion/react": "^11.11.4",
-        "@emotion/styled": "^11.11.0",
-        "@fortawesome/fontawesome-free": "^6.4.2",
-        "@fortawesome/fontawesome-svg-core": "^6.5.1",
-        "@fortawesome/free-brands-svg-icons": "^6.5.1",
-        "@fortawesome/free-regular-svg-icons": "^6.4.2",
-        "@fortawesome/free-solid-svg-icons": "^6.5.1",
+        "@emotion/styled": "^11.11.5",
+        "@fortawesome/fontawesome-free": "^6.5.2",
+        "@fortawesome/fontawesome-svg-core": "^6.5.2",
+        "@fortawesome/free-brands-svg-icons": "^6.5.2",
+        "@fortawesome/free-regular-svg-icons": "^6.5.2",
+        "@fortawesome/free-solid-svg-icons": "^6.5.2",
         "@fortawesome/react-fontawesome": "^0.2.0",
         "@jackstenglein/chess": "^1.0.64",
         "@jackstenglein/mui-mentions": "^0.1.6",
-        "@mui/base": "^5.0.0-beta.37",
+        "@mui/base": "^5.0.0-beta.40",
         "@mui/icons-material": "^5.15.11",
         "@mui/lab": "^5.0.0-alpha.166",
         "@mui/material": "^5.15.11",
-        "@mui/x-data-grid-pro": "^6.19.6",
-        "@mui/x-date-pickers": "^6.19.3",
-        "@testing-library/jest-dom": "^6.4.2",
-        "@testing-library/react": "^14.2.1",
-        "@testing-library/user-event": "^14.4.3",
-        "@types/node": "^20.11.24",
-        "@types/react": "^18.2.61",
-        "@types/react-dom": "^18.2.19",
+        "@mui/x-data-grid-pro": "^6.19.10",
+        "@mui/x-date-pickers": "^6.19.9",
+        "@svgr/core": "^8.1.0",
+        "@svgr/plugin-jsx": "^8.1.0",
+        "@types/node": "^20.12.7",
+        "@types/react": "^18.2.75",
+        "@types/react-dom": "^18.2.24",
+        "@vitejs/plugin-react": "^4.2.1",
         "@widgetbot/react-embed": "^1.9.0",
-        "aws-amplify": "^5.3.10",
+        "aws-amplify": "^5.3.18",
+        "axios": "^1.6.8",
+        "chart.js": "^4.4.2",
```

Comment:
We can remove chart.js right? You were going to replace react-charts with this but then didn't need to?